features/command.feature

```diff
@@ -719,3 +719,114 @@ Feature: WP-CLI Commands
       bar
       """
     And STDERR should be empty
+
+  @bootstrap
```

Comment:
What if we were to create a `feature/bootstrap.feature` file? I think breaking things apart is typically easier than using a bunch of tag annotations.